test/Microsoft.Diagnostics.EventFlow.Inputs.Tests/EventSourceInputTests.cs

```diff
@@ -183,6 +185,100 @@ public void CapturesEventsFromSourcesIdentifiedByNamePrefix()
             }
         }
 
+        [Fact]
+        public async Task CapturesEventCountersFromEventSource()
+        {
+            var healthReporterMock = new Mock<IHealthReporter>();
+
+            var inputConfiguration = new List<EventSourceConfiguration>();
+            inputConfiguration.Add(new EventSourceConfiguration()
+            {
+                ProviderName = "EventSourceInput-TestEventSource",
+                EventCountersSamplingInterval = 1
+            });
+
+            var eventSourceInput = new EventSourceInput(inputConfiguration, healthReporterMock.Object);
+            eventSourceInput.Activate();
+
+            var testTaskCompletionSource = new TaskCompletionSource<EventData>();
+
+            var observer = new Mock<IObserver<EventData>>();
+            observer.Setup(o => o.OnNext(It.IsAny<EventData>())).Callback<EventData>(eventData =>
+            {
+                testTaskCompletionSource.TrySetResult(eventData);
+            });
+
+            using (eventSourceInput.Subscribe(observer.Object))
+            {
+                EventSourceInputTestSource.Log.ReportTestMetric(5);
+                EventSourceInputTestSource.Log.ReportTestMetric(1);
+
+                var firstTaskToComplete = await Task.WhenAny(
+                    testTaskCompletionSource.Task,
+                    Task.Delay(TimeSpan.FromSeconds(1.5))
```

Comment:
Some of the machines used for testing can be highly loaded and/or very slow, so please make the timeout larger, say, 3 seconds.

Reply:
(applies to other tests as well)